index.html

```diff
@@ -30,16 +39,16 @@ <h2>The Past</h2>
             <h2>Why Did It Have To Be Boxes...</h2>
             
             <div class="boxes">
-                <div class="box">Box 1</div>
-                <div class="box">Box 2</div>
-                <div class="box">Box 3</div>
-                <div class="box">Box 4</div>
-                <div class="box">Box 5</div>
-                <div class="box">Box 6</div>
-                <div class="box">Box 7</div>
-                <div class="box">Box 8</div>
-                <div class="box">Box 9</div>
-                <div class="box">Box 10</div>
+                <div class="box" id="box1">Box 1</div>
+                <div class="box" id="box2">Box 2</div>
+                <div class="box" id="box3">Box 3</div>
+                <div class="box" id="box4">Box 4</div>
+                <div class="box" id="box5">Box 5</div>
+                <div class="box" id="box6">Box 6</div>
+                <div class="box" id="box7">Box 7</div>
+                <div class="box" id="box8">Box 8</div>
+                <div class="box" id="box9">Box 9</div>
+                <div class="box" id="box10">Box 10</div>
             </div>
```

Comment:
Great work an alternative to adding an id is the :nth-child css selectorhttps://developer.mozilla.org/en-US/docs/Web/CSS/:nth-child